conda_env.yml

```diff
@@ -4,14 +4,14 @@ channels:
 - ooi
 - conda-forge
 dependencies:
-- cython=0.25.2=py27_0
-- ipython=5.3.0=py27_0
+- cython=0.25.2
+- ipython=5.3.0
 - matplotlib
-- nose=1.3.7=py27_1
-- numexpr=2.6.2=np112py27_0
-- numpy=1.12
-- python<3
-- scipy
+- nose=1.3.7
+- numexpr=2.6.2
+- numpy=1.16
+- python=2.7
+- scipy=1.2.1
```

Comment:
Did you need to hard-specify scipy=1.2.1?

Reply:
Not necessarily, but numpy and scipy are large libraries that sometimes make breaking changes (e.g. like from numpy=1.12 to numpy=1.16), so I thought it safer to lock scipy down to something I knew was working.